code/core/src/components/components/AriaTabs/AriaTabList.stories.tsx

```diff
@@ -0,0 +1,50 @@
+import preview from '../../../../../.storybook/preview';
+import { AriaTabList } from './AriaTabList';
+import type { TabProps } from './AriaTabs';
+import { useAriaTabListState } from './AriaTabs';
+
+const DEFAULT_TABS: TabProps[] = [
+  { id: 'tab1', title: 'Tab 1', children: () => <div>Content for Tab 1</div> },
+  { id: 'tab2', title: 'Tab 2', children: () => <div>Content for Tab 2</div> },
+  { id: 'tab3', title: 'Tab 3', children: () => <div>Content for Tab 3</div> },
+];
+
+const meta = preview.meta({
+  title: 'AriaTabList',
+  component: AriaTabList,
+  args: {
+    tabs: DEFAULT_TABS,
+    state: undefined,
```

Comment:
Getting TS errors on this because tabs aren't an arg. I need help to figure out how to pass that data around and customise it per story (globals?).